server/src/main/java/io/kroki/server/Server.java

```diff
@@ -90,11 +92,21 @@ static void start(Vertx vertx, JsonObject config, Handler<AsyncResult<HttpServer
     BodyHandler bodyHandler = BodyHandler.create(false).setBodyLimit(config.getLong("KROKI_BODY_LIMIT", BodyHandler.DEFAULT_BODY_LIMIT));
 
     // CORS
+    // CORS Headers
     Set<String> allowedHeaders = new HashSet<>();
     allowedHeaders.add("Access-Control-Allow-Origin");
     allowedHeaders.add("Origin");
     allowedHeaders.add("Content-Type");
     allowedHeaders.add("Accept");
+    // Set additional Headers provided by environment variable
+    String envHeadersVar = System.getenv("ALLOWED_HEADERS");
```

Comment:
Please use the `config` object.
I think the option name should be `KROKI_CORS_ALLOWED_HEADERS`.

Reply:
We are using [Vert.x Config](https://vertx.io/docs/vertx-config/java/) which can be configured using environment variables or a configuration file.

So, if I set `KROKI_BODY_LIMIT=1234`, it will be available in the `config` object. Take a look at line 92:


```java
config.getLong("KROKI_BODY_LIMIT", BodyHandler.DEFAULT_BODY_LIMIT)
```

So it should be:

```suggestion
    String envHeadersVar = config.getString("KROKI_CORS_ALLOWED_HEADERS");
```

Reply:
Ahh ok understood, I changed it accordingly. Thanks.